src/api/accounts/businesses.md

```diff
@@ -0,0 +1,128 @@
+---
+layout:         default
+grand_parent:   API Reference
+parent:         Accounts
+title:          Businesses
+permalink:      /api/businesses
+---
+
+# Businesses
+{:.no_toc}
+
+A Business represents a company registered with the New Zealand Companies Office.
+
+A Business is associated with a single [Account][].
+
+## Contents
+{:.no_toc .text-delta}
+
+* TOC
+{:toc}
+
+## Models
+
+### Business
+
+|     Field     |        Type        |                    Description                     |
+| :------------ | :----------------- | :------------------------------------------------- |
+| id            | String             | The unique identifier.                             |
+| accountId     | String             | The Centrapay accountId.                           |
+| nzbn          | String             | The unique NZBN identifier.                        |
+| name          | String             | Trading name recorded in the Companies Register.   |
+| companyNumber | String             | Company number recorded in the Companies Register. |
+| createdAt     | {% dt Timestamp %} | When the Business was created.                     |
+| modifiedAt    | {% dt Timestamp %} | When the Business was updated.                     |
+
+## Operations
+
+### Create a Business
+
+{% reqspec %}
+  POST '/api/businesses'
+  auth 'api-key'
+  body ({
+    nzbn: '9429046246448'
+  })
+{% endreqspec %}
+
+{% h4 Required Fields %}
+
+| Field |  Type  |         Description         |
+| :---- | :----- | :-------------------------- |
+| nzbn  | String | The unique NZBN identifier. |
+
+
+{% h4 Optional Fields %}
+
+|   Field   |  Type  |       Description        |
+| :-------- | :----- | :----------------------- |
+| accountId | String | The Centrapay accountId. |
+
+{% warning If accountId is not provided when creating a business, then a new
+account will be created and associated to the business. %}
+
```

Comment:
I think we need some Errors here when:

* An accountId isn't an org account
* The NZBN cannot be found

Reply:
Added errors for both cases :)